code/simulator/renderscene.h

```diff
@@ -62,6 +62,11 @@ protected Q_SLOTS:
      */
     const std::vector<Solenoid> *solenoids() const override;
 
+    /**
+     * @return a pointer to the Sam instance.
+     */
+    const Sam *sam() const override;
```

Comment:
NIT: Sam should be in all caps, and I dont like just "Sam" either. Maybe change it to `SAMGraphics` or `SAMRobot` or whatever helps with understanding the the Sam class represents.